### DISCUSSION TITLE
Topbar search should not be contextual

### DISCUSSION THREAD

**Discussion by alexec** [community/community#33526] [Other Feature Feedback, Questions, & Ideas]
I always want to "search all of github", but it does not do that. It searches the repo, this behavour is very unclear. Either

(a) Make the context of the search much much clearer.
(b) Do not search in context.

**Comment by CaedenPH** [comment]
![image](https://user-images.githubusercontent.com/23421705/191116972-95c3d7e8-b7bb-4c86-bde0-60b4b115e9e9.png)

This is pretty user friendly and self explanatory, what do you propose instead?

**Reply by alexec** [reply]
Really? You have better eyesight that me to read such small dark grey text on a black background.
People see things differently, but I don't think you could describe this as "user friendly" and certainly not as accessible.

**Reply by CaedenPH** [reply]
> Really? You have better eyesight that me to read such small dark grey text on a black background. People see things differently, but I don't think you could describe this as "user friendly" and certainly not as accessible.

You can also use `ctrl + k`, `ctrl + alt + k`, `ctrl + shift + k` to open search mode which is much more extensive

**Reply by CaedenPH** [reply]
> > Really? You have better eyesight that me to read such small dark grey text on a black background. People see things differently, but I don't think you could describe this as "user friendly" and certainly not as accessible.
> 
> You can also use `ctrl + k`, `ctrl + alt + k`, `ctrl + shift + k` to open search mode which is much more extensive

Does this suit your needs?
